spk/flexget/Makefile

```diff
@@ -1,16 +1,19 @@
 SPK_NAME = flexget
-SPK_VERS = 2.9.14
-SPK_REV = 6
+SPK_VERS = 3.1.77
+SPK_REV = 7
 SPK_ICON = src/flexget.png
 
+REQUIRED_DSM = 6.0
+
+BUILD_DEPENDS = cross/python3 cross/setuptools cross/pip cross/wheel
 WHEELS = src/requirements.txt
-SPK_DEPENDS = "python>=2.7.8-11"
+SPK_DEPENDS = "python3>=3.7.7"
 
 MAINTAINER = Diaoul
 DESCRIPTION = FlexGet is a multipurpose automation tool for content like torrents, nzbs, podcasts, comics, series, movies, etc. It can use different kinds of sources like RSS-feeds, html pages, csv files, search engines and there are even plugins for sites that do not provide any kind of useful feeds.
 DISPLAY_NAME = FlexGet
-BETA = 1
-CHANGELOG = "Update Flexget to version 2.9.14"
+BETA = 0
+CHANGELOG = "Update FlexGet to version 3.1.77"
 
 HOMEPAGE   = http://flexget.com/
 LICENSE    = GPL
```

Comment:
when I look in the official flexget github repo, it shows the license is MIT (https://github.com/Flexget/Flexget)
can you please verify?

Reply:
You are right.